external/storm-kafka-monitor/src/main/java/org/apache/storm/kafka/monitor/KafkaOffsetLagUtil.java

```diff
@@ -89,6 +89,10 @@ public static void main (String args[]) {
                     printUsageAndExit(options, OPTION_ZK_SERVERS_LONG + " and " + OPTION_ZK_COMMITTED_NODE_LONG + " are required  with " +
                             OPTION_OLD_CONSUMER_LONG);
                 }
+                String zkNode = commandLine.getOptionValue(OPTION_ZK_COMMITTED_NODE_LONG);
```

Comment:
we can just remove the check since `curatorFramework.checkExists().forPath(zkPath)` will do the same check for zkpath. Thanks.